src/modules/htmlElements.js

```diff
@@ -0,0 +1,3 @@
+const ul = document.querySelector('body ul');
+const input = document.getElementById('add-input');
```

Comment:
- [ ] Please Make sure a function for adding a new task (**add a new element to the array**) is well implemented and works fine.